README.md

```diff
@@ -25,6 +25,49 @@ If you want to download regularly updated builds for Android, Windows x86 and x6
 
 For game compatibility, see [community compatibility feedback](https://report.ppsspp.org/games).
 
+What's new in 1.16
+==================
+- RetroAchievements support ([#17589], [#17631], many more). See [RetroAchievements on ppsspp.org](https://www.ppsspp.org/docs/reference/retro-achievements).
+- New JIT backends:
+  - RISC-V, based on IR, by unknownbrackets
```

Comment:
Not sure if it matters who, if we say it here we should note fp64 under vrnd at least.

-[Unknown]

Reply:
Just thought I'd add some extra credit since it's such a significant chunk of really good work. Although I suppose it sets a messy-looking precedent :) But yeah, i think I'll add fp64 for vrnd.

Reply:
You should totally add him to the credit screen too, as if someone ever read that :P